scripts/appconnect_cli.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env sentry exec
+"""Dumb CLI to experiment with our App Store Connect functionality.
+
+It contains several useful tools to manipulate the database during development.  Exactly
+what it does can be easily changed based on new needs though, nothing is set in stone.
+
+Most actions assume you have only one App Store Connect source and will simply pick the
+first one found.
+"""
+import sys
+from pprint import pprint
+
+from sentry.lang.native.appconnect import SYMBOL_SOURCES_PROP_NAME
+from sentry.models import AppConnectBuild, Project
+from sentry.tasks import app_store_connect
+from sentry.utils import json
+
+PROJECT_ID = 2
+
+
+def main(argv):
+    if argv[0] == "dump-cfg":
```

Comment:
You're welcome to land, however, this script could be better written with [argparse](https://docs.python.org/3.6/library/argparse.html) or [click](https://pypi.org/project/click/).

Reply:
yeah, I know 😄 for now this is not a general purpose script for operators though so i thought full on arg parsing was a bit overkill.  thanks for approving though.